tests/hypertext/HTMLLintBearTest.py

```diff
@@ -27,3 +27,10 @@
     invalid_files=(test_file,),
     settings={'htmllint_ignore': 'quotation'},
     tempfile_kwargs={'suffix': '.html'})
+
+HTMLLintBearDisableCheckerTest = verify_local_bear(
+    HTMLLintBear,
+    valid_files=(test_file,),
+    invalid_files=(),
+    settings={'optional_tag': 'off'},
```

Comment:
That price of code is actually informing the user that they provided invalid disable arguments.